### ISSUE TITLE
Field naming: 'username' and 'nickname'

### ISSUE THREAD

**Issue by RXWE** [swsnu/swpp2019-team6#14]
Design and planning 문서를 작성할 때, model과 frontend design에서
* username: '유저 아이디'라고 칭할 수 있는 부분, 여기서는 유저가 등록한 이메일
* nickname: 유저가 설정한, 서비스 이용 시 보이는 유저의 이름(인스타그램 유저명과 같음)

으로 정의했습니다. 이렇게 알고 코드를 작성했는데, 두 field가 의미하는 바가 비슷해서 헷갈리더라구요.
그래서 이 두 필드명을 **email / username** 이런 식으로 바꾸면 어떨까요? 가입할 때 이메일 정보를 이용하자는 것은 정해졌고, 유저가 설정한 유저의 이름을 username으로 명명하는 게 더 명확하지 않을까 싶었습니다!

**Comment by deploy-soon** [comment]
네 이름이 헷갈린다면 바꾸는 것도 좋을 것 같아요. 다만 프론트는 괜찮은데 백에서는 django user model에서 아이디와 비밀번호를 암묵적으로 username과 password라고 부르더라고요. 만약 nickname을 username으로 명명하면 더 헷갈릴 수도 있을 것 같아요.

프론트랑 백이랑 같을 필요는 없지만 통일되면 나중에 커뮤니케이션이 잘 될 것 같아서요.. 혹시 입력하는 아이디를 email로 nickname을 그대로 nickname으로 두는건 어떨까요

**Comment by RXWE** [comment]
백에서 아이디로 username을 사용하는 줄은 몰랐네요! 그러면 말씀대로 nickname은 그대로 두고, 유저가 입력하는 건 email로 바꾸면 혼동이 없을 것 같아요. 다른 분들도 괜찮으신가요??

**Comment by deploy-soon** [comment]
네 백에서도 email로 사용하도록 할게요!

**Comment by jangdonghae** [comment]
백에서도 email로 사용한다는게 어떻게 쓰는건지 알려주실 수 있나요?

**Comment by deploy-soon** [comment]
개발중인 branch의 [코드](https://github.com/swsnu/swpp2019-team6/commit/ebf73c431bef68c76b0a7815f7b168a6a6710ba6#diff-305062d7e9e82f92cdc26bb606579ad5L7-R15) 참고해주시면 model의 username을 email로 지정하고 User의 `USERNAME_FIELD`를 email로 지정한 것을 확인할 수 있습니다.

이후 authenticate 모델 혹은 serializer에서 `USERNAME_FIELD`을 통해 field를 확인하고 수정하기 때문에 이런식으로 구현하였습니다. ([관련코드](https://github.com/swsnu/swpp2019-team6/commit/ebf73c431bef68c76b0a7815f7b168a6a6710ba6#diff-2f78c2789ef6028f7cebac2d1f9b3c41L47-L53))
